packages/genesys-spark-components/src/genesys-spark-utils/intl.ts

```diff
@@ -84,11 +84,23 @@ export function determineDisplayLocale(
  */
 function browserHasRegionData(localeString: string): boolean {
   return (
-    localeString.length == 2 &&
-    navigator.language?.startsWith(`${localeString}-`)
+    browserLocaleOverride(localeString) ||
+    (localeString.length == 2 &&
+      navigator.language?.startsWith(`${localeString}-`))
   );
 }
 
+function browserLocaleOverride(localeString: string): boolean {
+  switch (localeString.toLowerCase()) {
+    case 'zh-cn':
+      return !!navigator.language?.toLowerCase().startsWith('zh-sg');
```

Comment:
Would  `return  navigator.language?.toLowerCase().startsWith('zh-sg') || false` work also ? I always find it hard to parse the double negation pattern 😆

Reply:
`Boolean(navigator.language?.toLowerCase().startsWith('zh-hk'))` is also an option :)